config/config.go

```diff
@@ -729,7 +735,7 @@ func (cfg *Configuration) AccountDefaultsJSON() json.RawMessage {
 	return cfg.accountDefaultsJSON
 }
 
-//Allows for protocol relative URL if scheme is empty
+// Allows for protocol relative URL if scheme is empty
```

Comment:
Nipick: If you're going to fix the comment, please change it to start with the function name, ie
`// GetBaseURL allows for...`